danndigital/debbie-portfolio-2021
# New contact form

## Description

- Removed email.js
- Added Netlify form. Followed [Netlify documentation](https://www.netlify.com/blog/2017/07/20/how-to-integrate-netlifys-form-handling-in-a-react-app/) to implement. 

## Related Issue

Closes #26 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

- From your terminal, pull down this branch with `git pull origin new-contact-form` and check that branch out with `git checkout new-contact-form`.
- Then `npm start` to launch the app.
- Try submitting without entering data, it should give error message.
- Complete form and submit data, you should get a success alert.
